src/test/java/redis/clients/jedis/modules/bloom/TDigestTest.java

```diff
@@ -0,0 +1,171 @@
+package redis.clients.jedis.modules.bloom;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Random;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import redis.clients.jedis.modules.RedisModuleCommandsTestBase;
+import redis.clients.jedis.util.KeyValue;
+
+public class TDigestTest extends RedisModuleCommandsTestBase {
+
+  private static final Random random = new Random();
+
+  @BeforeClass
+  public static void prepare() {
+    RedisModuleCommandsTestBase.prepare();
+  }
+//
+//  @AfterClass
+//  public static void tearDown() {
+////    RedisModuleCommandsTestBase.tearDown();
+//  }
+
+  private void assertMergedUnmergedNodes(String key, int mergedNodes, int unmergedNodes) {
+    Map<String, Object> info = client.tdigestInfo(key);
+    assertEquals(Long.valueOf(mergedNodes), info.get("Merged nodes"));
+    assertEquals(Long.valueOf(unmergedNodes), info.get("Unmerged nodes"));
+  }
+
+  private void assertTotalWeight(String key, double totalWeight) {
+    Map<String, Object> info = client.tdigestInfo(key);
+    assertEquals(totalWeight, Double.parseDouble((String) info.get("Merged weight"))
+        + Double.parseDouble((String) info.get("Unmerged weight")), 0.01);
```

Comment:
Why there is 0.01 in the assertEquals?
the total weight should not always be always equal to 0.01, no?

Reply:
@shacharPash I don't get what your exact question is but here is my first attempt to answer:

JUnit, the library we based our tests upon, deprecated [assertEquals(double, double)](https://junit.org/junit4/javadoc/4.13/org/junit/Assert.html#assertEquals(double,%20double)) and suggested to use [assertEquals(double, double, double)](https://junit.org/junit4/javadoc/4.13/org/junit/Assert.html#assertEquals(double,%20double,%20double)).